src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -27,57 +28,51 @@ def __init__(self, configuration):
             configuration['hookId'],
         )
 
-        # Save phabricator connection data
+        # Connect to Phabricator API
         assert 'phabricator_url' in configuration
         assert 'phabricator_token' in configuration
-        self.api_url = configuration['phabricator_url']
-        self.api_token = configuration['phabricator_token']
+        self.api = PhabricatorAPI(
+            api_key=configuration['phabricator_token'],
+            url=configuration['phabricator_url'],
+        )
+
+        # List enabled repositories
+        enabled = configuration.get('repositories', ['mozilla-central', ])
```

Comment:
By default we want to enable `autoland` and `mozilla-inbound` too

Reply:
I can't find those on https://phabricator.services.mozilla.com/api/diffusion.repository.search :disappointed: